src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,61 @@
 'use strict';
 
+'use strict';
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
The `hidden` property should only be defined in the `Herbivore` class, as it's specific to that class and not all `Animal` instances.